src/runtime_src/core/pcie/emulation/cpu_em/generic_pcie_hal2/shim.h

```diff
@@ -494,6 +502,7 @@ namespace xclcpuemhal2 {
       exec_core* mCore;
       SWScheduler* mSWSch;
       bool mIsKdsSwEmu;
+      std::atomic_bool mIsDeviceProcessStarted;
```

Comment:
Why do I have to list every instance of same changes?  
```suggestion
      std::atomic<bool> mIsDeviceProcessStarted;
```